--- a/configure	2021-07-24 16:07:19.677207112 -0500
+++ b/configure	2021-07-24 16:09:42.547992756 -0500
@@ -732,9 +732,9 @@
 #
 # Initializations.
 #
-ac_default_prefix=/usr/local
+ac_default_prefix=/
 ac_clean_files=
-ac_config_libobj_dir=.
+ac_config_libobj_dir=/usr/
 LIBOBJS=
 cross_compiling=no
 subdirs=
@@ -786,7 +786,7 @@
 # include <unistd.h>
 #endif"
 
-ac_default_prefix=/usr/local
+ac_default_prefix=/
 ac_subst_vars='am__EXEEXT_FALSE
 am__EXEEXT_TRUE
 LTLIBOBJS
@@ -960,23 +960,23 @@
 # by default will actually change.
 # Use braces instead of parens because sh, perl, etc. also accept them.
 # (The list follows the same order as the GNU Coding Standards.)
-bindir='${exec_prefix}/bin'
-sbindir='${exec_prefix}/sbin'
-libexecdir='${exec_prefix}/libexec'
-datarootdir='${prefix}/share'
+bindir='${prefix}/bin'
+sbindir='${prefix}/sbin'
+libexecdir='/usr/libexec/'
+datarootdir='/usr/share'
 datadir='${datarootdir}'
 sysconfdir='${prefix}/etc'
 sharedstatedir='${prefix}/com'
 localstatedir='${prefix}/var'
-includedir='${prefix}/include'
+includedir='/usr/include'
 oldincludedir='/usr/include'
-docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+docdir='${datarootdir}/doc/${P}'
 infodir='${datarootdir}/info'
 htmldir='${docdir}'
 dvidir='${docdir}'
 pdfdir='${docdir}'
 psdir='${docdir}'
-libdir='${exec_prefix}/lib'
+libdir='/usr/$(get_libdir)'
 localedir='${datarootdir}/locale'
 mandir='${datarootdir}/man'
 
